server/src/Diff.ts

```diff
@@ -93,6 +128,41 @@ export class Diff {
     return oldNumber;
   }
 
+  static getNewNumber(oldString: string, newString: string, oldNumber: number, countFromDiff: (str: string) => number): number | undefined {
```

Comment:
Is the algorithm different from existing function or it is possible to extract any common part to separate function?

Reply:
Seems not.